content/docs/reference/microcontrollers/machine/arduino-mega1280.md

```diff
@@ -594,6 +596,21 @@ WriteByte is a no-op: the null serial doesn't write bytes.
 
 
 
+## type PDMConfig
+
+```go
+type PDMConfig struct {
+	Stereo	bool
+	DIN	Pin
+	CLK	Pin
+}
+```
+
```

Comment:
Lots of blank lines here.  Assuming this is generated so we can fix up the generator later.

Reply:
Yeah this is autogenerated and IIRC isn't visible when rendered. Feel free to send a fix though ;)